db/migrate/20160610124506_add_lives_on_to_middleware_server.rb

```diff
@@ -0,0 +1,11 @@
+class AddLivesOnToMiddlewareServer < ActiveRecord::Migration[5.0]
```

Comment:
Isn't it that reversible operations are supposed to be done with `up` and `down`, whereas non-reversible operations are done via `change`?